operator/go.mod

```diff
@@ -487,7 +487,7 @@ replace (
 
 	// Roughly equivalent to redpanda chart version v5.9.19. pkg, connectors, and console are inherited from redpanda.
 	// TODO it may behoove us to split gotohelm into it's own module out of pkg so the operator doesn't have to follow redpanda.
-	github.com/redpanda-data/redpanda-operator/charts/redpanda => github.com/redpanda-data/redpanda-operator/charts/redpanda v0.0.0-20250207141022-1388ec6c6b63
+	// github.com/redpanda-data/redpanda-operator/charts/redpanda => ../charts/redpanda
```

Comment:
@chrisseto rather than leveraging a replace directive, I just directly modified the version we ref in the `require` directive above, pointing it at https://github.com/redpanda-data/redpanda-operator/commit/c445c9211ec5fe723c40fd0b5ff82de3822567cc

Reply:
That works too! The only reason I was using replace was to side step potential issues with `go mod tidy`. As long as it's stable it's good to go. ~Small chance that the go mod linter complains as it~ heh build failed right as I was typing...